FireboltNETSDK/Client/FireboltCommand.cs

```diff
@@ -268,6 +268,25 @@ private string GetServerSideQueryParameters()
             );
         }
 
+        private async Task<StreamReader> ExecuteStreamingCommandAsync(string commandText, CancellationToken cancellationToken = default)
```

Comment:
I see a couple of issues with this method
1. It duplicates a lot of code from `ExecuteCommandAsync`
2. It doesn't respect the `CommandTimeoutMillis` property. For `ExecuteCommandAsync`, we wrap it with `ExecuteCommandAsyncWithCommandTimeout`, which adds timeout handling. I believe we should do the same here